router/router.go

```diff
@@ -672,16 +672,29 @@ func (worker *workerT) processDestinationJobs() {
 							} else {
 								// stat start
 								pkgLogger.Debugf(`responseTransform status :%v, %s`, worker.rt.transformerProxy, worker.rt.destName)
-								sendCtx, cancel := context.WithTimeout(ctx, worker.rt.netClientTimeout)
-								defer cancel()
+								var sendCtx context.Context
+								var cancel context.CancelFunc
```

Comment:
Don't see any need for these to be declared beforehands
```suggestion
```

Reply:
sendCtx, cancel are being used for both SendPost(Router -> destination), ProxyRequest(Router -> proxy). So I think we should declare it beforehand. Let me know your thoughts

Reply:
Neither of them makes sense outside the if else block, thus each block can simply define its own same variables